recipe/meta.yaml

```diff
@@ -15,83 +17,188 @@ source:
   build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - numpy                                  # [build_platform != target_platform]
     - {{ compiler('cxx') }}
-    - {{ stdlib("c") }}
+    - {{ stdlib('c') }}
+    - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+    - numpy                               # [build_platform != target_platform]
+    - postgresql                          # [build_platform != target_platform]
+    - python                              # [build_platform != target_platform]
     - cmake
-    - jom  # [win]
+    - jom   # [win]
     - make  # [unix]
     - pkg-config
   host:
-    - libboost-devel
-    - libboost-python-devel
     - cairo
     - eigen
     - freetype
-    - python
+    - libboost-devel
+    - libboost-python-devel
     - numpy
-    - pillow
     - pandas <2.2
+    - pillow
     - pip
+    - postgresql
+    - python
     - setuptools
     - setuptools_scm >=8
-    # For rdkit-stubs
-    - pybind11-stubgen
-  run:
-    - cairo
-    - python
-    - pillow
-    - pandas
-    - {{ pin_compatible('numpy') }}
-    - pycairo
-    - matplotlib-base
-    - sqlalchemy
-    - reportlab
-
-test:
-  commands:
-    - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
-  imports:
-    - rdkit
-    - rdkit.Avalon
-    - rdkit.Chem
-    - rdkit.Chem.AllChem
-    - rdkit.Chem.rdFreeSASA
-    - rdkit.DataManip
-    - rdkit.Dbase
-    - rdkit.DistanceGeometry
-    - rdkit.ForceField
-    - rdkit.Geometry
-    - rdkit.ML
-    - rdkit.Numerics
-    - rdkit.SimDivFilters
-    - rdkit.VLib
-    - rdkit.VLib.NodeLib
-    - rdkit.Chem.PandasTools
 
 outputs:
-  - name: rdkit
-  - name: rdkit-dev
-    script: install_rdkit_dev.bat  # [win]
+  - name: librdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    requirements:
+      build:
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - matplotlib-base
+        - numpy
+        - pandas <2.2
+        - pillow
+        - pycairo
+        - python
+        - reportlab
+        - sqlalchemy
+      run:
+        - cairo
+        - eigen
+        - freetype
+        # NOTE(skearnes): The `host` dependencies above are only listed so that this pin is correctly resolved. Without
+        # them (or some subset that I haven't narrowed to yet), libboost may be resolved to a higher version than was
+        # used for the build in build.sh.
+        - {{ pin_compatible('libboost') }}
```

Comment:
Hm, libboost-devel should be pushing its own [run_exports](https://github.com/conda-forge/boost-feedstock/blob/67119c573017e5c534063abf6434080d915c90a4/recipe/meta.yaml#L172-L174), not sure why we need this.

Reply:
I think this is because libboost has a max_pin of `x.x`, and this fails if I get libboost 1.85 instead of 1.84 (which happens if I remove those host reqs).

Reply:
See https://github.com/conda-forge/boost-feedstock/blob/main/recipe/meta.yaml#L174

Reply:
But yeah I don't understand why this is happening either; I think I need to connect the pin to the metapackage build version somehow?

Reply:
Huh; if I force the top-level host boost deps to >1.84 I can get things to build with 1.85. I have to specify the pin for both libboost-devel and libboost-python-devel though, which seems odd.

Reply:
OK now this is getting weird: if I specify >=1.84 I get 1.85 (instead of 1.84 without any pin)

Reply:
I'm getting mismatches in the python subpackage now; I'm just going to fix the boost version and call it good. Then I can remove all of those extra host deps.